samples/nrf9160/aws_fota/Kconfig

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "mqtt fota"
+
+config DOWNLOAD_HOST
+	string "Download host name"
+
+config DOWNLOAD_FILE
+	string "The file to download"
+
+config MQTT_PUB_TOPIC
```

Comment:
This file needs a cleanup, please remove unused configs.

Reply:
Tried to address this in the latest commit.